### PR TITLE
API-10657: add info about bugfix of groups limits of bots in v3.4+ to…

### PR DESCRIPTION
**Pull request by bkubiak**
# Links
- [Jira](https://livechatinc.atlassian.net/browse/API-10657)

# Description

This PR introduces a bugfix in the changelog for the first time. I'm not sure about the format and description 🤔 I think we should inform users about the date because the bugfix was introduced to the released version of API. Also, I'm not sure about the icon - if this is self-descriptive 🤔.

### CODE REVIEW

File: src/pages/management/changelog/index.mdx [hunk 1]
```diff
@@ -46,6 +46,7 @@ The developer preview version provides a preview of the upcoming changes to the
 ### Bots
 
 - Added support for the `supervisor` priority for the `groups[].priority` parameter. This applies to all **v3.x** versions of *the Configuration API*.
+- 🛠️ (2022-04-04): Increased possible size of `groups[]` parameter when adding or updating a bot.
```
Comment (gluszczyk) [inline src/pages/management/changelog/index.mdx:49]: possible -> max, methinks

Comment (bkubiak) [reply src/pages/management/changelog/index.mdx:49]: I will update when @ale-kacperczyk reviews this, especially when you guys comment about the date and format of the bugfix.

**Review by ale-kacperczyk — COMMENTED** [review]
https://6245b71d208f936936fef6a8--livechat-public-docs.netlify.app/management/changelog

File: src/pages/management/changelog/index.mdx (hunk 1 above)
Comment (ale-kacperczyk) [inline src/pages/management/changelog/index.mdx:49]:
```suggestion
- 🛠️ (2022-04-04): Increased the maximum size of the `groups[]` parameter when adding or updating a bot.
```

**Review by ale-kacperczyk — APPROVED** [review]

**Review by gluszczyk — APPROVED** [review]

**Pull request merged**